src/Desktop/ColorContrastAnalyzer/RowResultV2Accumulator.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
The new approach is more performance-intensive. Have you tried running it on larger bitmaps? I'm wondering what would happen if we focused on an element with a very large bounding rectangle from AI-Windows; it would be nice to understand how this scales with bitmap size to avoid app stutters.

Reply:
I don't yet have numbers, but in practice it's perhaps less of a difference than it might seem. For a large bitmap (an entire app, for instance), you're likely to have a lot of controls and not be able to get quickly to a High confidence number. If this happens, the old code will scan about 1/12 of the scanlines (by default) before giving up.

I definitely want to benchmark this (and possibly see if using the `System.Drawing.Color` _struct_ instead of the `ColorContrastAnalyzer.Color` _class_ will improve our perf), but this PR is already really large just getting the functionality in--perf measurements and tuning feel like they belong in a separate PR